content/information-distribution/article.md

```diff
@@ -29,27 +29,38 @@ If the sender is not a source user, the article **should** be ignored. If the
-| `custom`            | `object` |      | Additional information for custom client implementations.                                                                                  |
+| Parameter           | Type              | Req. | Description                                                                                                                                |
+|:--------------------|:------------------|:----:|:-------------------------------------------------------------------------------------------------------------------------------------------|
+| `title`             | `string`          |  x   | Article's headline.                                                                                                                        |
+| `href`              | `string`          |      | URL of the article's original post (in case the article was sent to Informo from an existing website).                                     |
+| `short_description` | `string`          |      | Short description or introduction to the article.                                                                                          |
+| `author`            | `string`          |      | Full name of the article's author (when a single Informo source aggregates multiple writers).                                              |
+| `thumbnail`         | `string`          |      | Preview image for the article. Must be a [`mxc://` URL](https://matrix.org/docs/spec/client_server/r0.4.0.html#id112).                     |
+| `date`              | `int`             |      | Timestamp in milliseconds of the article's publication. If not provided clients should fall back to the Matrix event's creation timestamp. |
+| `content`           | `string`          |  x   | Article HTML content. The HTML **must** be sanitized before being displayed in a client.                                                   |
+| `media_sigs`        | `MediaSignatures` |      | Cryptographic signatures of the media embedded in the article.                                                                             |
+| `custom`            | `object`          |      | Additional information for custom client implementations.                                                                                  |
+
+Where:
+
+* `MediaSignatures` is a map associating a [`mxc://`
+  URL](https://matrix.org/docs/spec/client_server/r0.4.0.html#id112) with the
+  signature of the media that's been uploaded at this URL. This signature
+  **must** be generated using the same algorithm and public key that were used
+  to sign the article's event. If this is not the case, or the signature can't
+  be verified, client implementations **must not** display the media to their
+  users. The `mxc://` URL **can** refer to any media included either in the
+  article's content or in any other property in the event's data (such as its
+  thumbnail).
```

Comment:
Maybe add a note saying every mxc URL **must** have an associated signature, even though it's implied by  "If this is not the case, or the signature can't be verified, client implementations **must not** display the media"